src/main/java/Storage.java

```diff
@@ -0,0 +1,88 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Storage class serves to save users' task lists to local machines' disk
+ * so that the previous run's list will be available to users for reference.
+ */
+public class Storage {
+
+    //public static String home = System.getProperty("src/data/duke.txt");
+    private static final String home = "data";
```

Comment:
Should the variable name be "HOME" with all uppercase letters since it is a constant?